addons/knowledge/views/knowledge_views.xml

```diff
@@ -0,0 +1,247 @@
+                <field name="favourite_count" invisible="1"/>
+                <field name="main_article_id"/>
+                <field name="last_edition_uid" widget="many2one_avatar_user"/>
+                <field name="last_edition_date"/>
+
+                <field name="owner_id" widget="many2one_avatar_user" groups="base.group_system"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="knowledge_article_view_kanban" model="ir.ui.view">
+        <field name="name">knowledge.article.view.kanban</field>
+        <field name="model">knowledge.article</field>
+        <field name="arch" type="xml">
+            <kanban class="o_knowledge_kanban_view">
+                <field name="icon" invisible="1"/>
+                <field name="is_locked" invisible="1"/>
+                <field name="user_can_write" invisible="1"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="oe_kanban_global_click">
+                            <div class="o_knowledge_kanban_card d-flex flex-column">
+                                <div class="d-flex align-items-center">
+                                    <div class="flex-grow-1 lead text-truncate">
+                                        <field name="display_name"/>
+                                    </div>
+                                    <field name="is_user_favourite" class="m-0" widget="boolean_favorite" nolabel="1" force_save="1"/>
+                                    <div attrs="{'invisible': ['|', ('is_locked', '=', False), ('user_can_write', '=', False)]}">
+                                        <i class="fa fa-fw fa-lock" title="locked"/>
+                                    </div>
+                                </div>
+                                <div class="flex-grow-1 text-muted">
+                                    <field name="category"/>
+                                </div>
+                                <div class="d-flex justify-content-end align-items-center">
+                                    <field name="activity_ids" widget="kanban_activity"/>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="knowledge_article_view_search" model="ir.ui.view">
+        <field name="name">knowledge.article.view.search</field>
+        <field name="model">knowledge.article</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="main_article_id"/>
+                <field name="last_edition_uid"/>
+
+                <filter string="Archived" name="active" domain="[('active', '=' ,False)]"/>
+                <filter name="favourites" string="My Favourites" domain="[('is_user_favourite', '=', True), ('user_has_access', '=', True)]"/>
```

Comment:
Not sure why you need to manually add "user_has_access" all the time.
Isn't it what the ACLs automatically do?